src/monattack.cpp

```diff
@@ -3375,12 +3375,19 @@ void mattack::taze( monster *z, Creature *target )
 {
     // It takes a while
     z->moves -= 200;
-    if( target == nullptr || target->uncanny_dodge() ) {
+    if( target == nullptr ) {
+        return;
+    }
+    if( target->uncanny_dodge() || dodge_check( z, target ) ) {
```

Comment:
`uncanny_dodge` already prints out a message, so it might be better to keep the original condition and have `dodge_check( z, target )` as its own conditional
```suggestion
    if( target == nullptr || target->uncanny_dodge() ) {
        return;
    }
    if( dodge_check( z, target ) ) {
```
Otherwise, assuming `uncanny_dodge` passes, it will print out something like:
```
"Time seems to slow down and you effortlessly dodge!"
"The {critter} tries to shock you, but you dodge."
```

Reply:
Ack, automerge was on so didn't get the chance to think up a fix, aaaa